### ISSUE TITLE
Bug: `isStrict` option in `no-unnecessary-act` only reports strict violations

### ISSUE THREAD

**Issue by zaicevas** [testing-library/eslint-plugin-testing-library#416]
With `isStrict: true` the `no-unnecessary-act` rule only reports mixed RTL+non RTL calls, but does not report `act` with only RTL calls, which is the behaviour without `isStrict` enabled.

```
      await act(async () => {
        userEvent.click(contactSupport) // not reported when isStrict is true
      })
```

I'll take a look at this one when I have the time.

**Comment by Belco90** [comment]
This should have been caught by some test 😞
